js/widgets/site-search.js

```diff
@@ -6,9 +6,9 @@ define([
     'dataset/typeahead',
     'reuse/typeahead',
     'organization/typeahead',
-    // 'user/typeahead',
+    'territory/typeahead',
```

Comment:
We may expose the feature availability in the raw.html page header like:
```html+jinja
<meta name="territory-enabled" content="{{ config.ACTIVATE_TERRITORIES }}">
```
and use it in config.js
```javascript
export default {
    territory_enabled: $('meta[name=territory-enabled]').attr('content').toLowerCase() === 'true'
};
```